core/memory_config.py

```
@@ -0,0 +1,131 @@
+RU: Модуль конфигурации для Memori - SQL-native memory engine для AI агентов.
+EN: Configuration module for Memori - SQL-native memory engine for AI agents.
+
+Memori автоматически сохраняет контекст взаимодействий и позволяет AI агентам
+запоминать предыдущие диалоги и предпочтения пользователей.
+"""
+
+import logging
+import os
+from typing import Optional
+
+from typing import Optional, TYPE_CHECKING, Any
```

Flake8 (7.3.0)

[error] redefinition of unused 'Optional' from line 13

(F811)